### ISSUE TITLE
Remember this?

### ISSUE THREAD

**Issue by nfeiglin** [boardofstudies/crossroads#1]
:books:

**Comment by paked** [comment]
The harrison and mark carry is real :)

**Comment by nfeiglin** [comment]
+1

On Mon, Jul 27, 2015 at 8:31 PM, Harrison Shoebridge <
notifications@github.com> wrote:

> The harrison and mark carry is real :)
> 
> On Mon, 27 Jul 2015 1:05 pm Nathan F notifications@github.com wrote:
> 
> > [image: :books:]
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/boardofstudies/crossroads/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/boardofstudies/crossroads/issues/1#issuecomment-125159407
> .

## 

Twitter https://twitter.com/nfeiglin | LinkedIn
https://www.linkedin.com/in/nfeiglin | Medium
https://medium.com/@nfeiglin

**Comment by paked** [comment]
> :+1: 

FTFY

**Comment by paked** [comment]
But in all seriousness, If we had all put _actual_ effort into it... We had a shot at getting a prize I reckon!

**Comment by nfeiglin** [comment]
I think there were too many voices trying to pull the product in opposing
directions (too many mentors, not enough devs?) that ended up paralysing
up. Anyway, it was fun. :shipit:

On Wed, Jul 29, 2015 at 9:11 AM, Harrison Shoebridge <
notifications@github.com> wrote:

> But in all seriousness, If we had all put _actual_ effort into it... We
> had a shot at getting a prize I reckon!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/boardofstudies/crossroads/issues/1#issuecomment-125778600
> .

## 

Twitter https://twitter.com/nfeiglin | LinkedIn
https://www.linkedin.com/in/nfeiglin | Medium
https://medium.com/@nfeiglin

**Comment by paked** [comment]
yeah, fair call. If we do it again, definitely designate one SINGLE person to actually manage the thing...

**Comment by nfeiglin** [comment]
We need a PM :wink:

**Comment by paked** [comment]
/:100:
